lib/simp/metadata/component.rb

```diff
@@ -202,7 +202,7 @@ def extract
       def locations
         # XXX: ToDo Allow manifest.yaml to override locations
         # XXX: ToDo Use primary_source and mirrors here if locations is empty
-        Simp::Metadata::Locations.new({ 'locations' => get_from_component['locations'], 'primary_source' => get_from_component['primary_source'], 'mirrors' => get_from_component['mirrors'] }, self)
+        Simp::Metadata::Locations.new({:locations => get_from_component['locations'], :primary_source => get_from_component['primary_source'], :mirrors => get_from_component['mirrors'] }, self)
```

Comment:
Just another note for consistency. Same as the first comment where this has 0 leading spaces but one trailing space before the close }.